build/defines.mak

```diff
@@ -59,6 +59,7 @@ DEFAULT_PY_FILES_TO_GENERATE := \
     unit_tests/_matchers.py \
     __init__.py \
     _converters.py \
+    _complextype.py \
```

Comment:
Alternatively, we could add this to `MODULE_FILES_TO_GENERATE` in nifake.mak and nirfsg.mak or filter it out of `MODULE_FILES_TO_GENERATE` for all other clients.

I'm not sure I want us to add this module to clients that don't use it.

Reply:
You could also argue that we're importing other unused modules to session.py, which is we have `# noqa: F401` for the enums import. The difference is that enums.py could potentially be used by session.py in the future with the addition of a new method, whereas some APIs will never use these complex types.

Reply:
If we're going to change it, I'd rather it be in this PR.
It'll greatly reduce the number of files in the PR, if we're only changing the generated output for nirfsg and nifake.